### PR TITLE
feat(core): add lollipop graph

### PR DESCRIPTION
**Pull request by theiliad**
image

### CODE REVIEW

**Comment by theiliad** [comment]
Needs a design review

**Comment by mjabbink** [comment]
👍

File: packages/core/src/components/graphs/lollipop.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+			.attr("x2", getXValue)
+			.attr("y1", mainYScale.range()[0])
+			.attr("y2", getYValue)
+			.attr("stroke", (d) =>
+				this.model.getFillColor(d[groupMapsTo], d[domainIdentifier], d)
+			)
+			.attr("opacity", 1);
+	}
+
+	// getRadiusScale(selection: Selection<any, any, any, any>) {
```
Comment (cal-smith) [inline packages/core/src/components/graphs/lollipop.ts:73]: Do we need to keep any of the commented code?

**Review by cal-smith — COMMENTED** [review]
One comment, looks good overall though 👍

**Comment by DanaPaslaru** [comment]
Hi, these are the design explorations for this chart: https://github.com/carbon-design-system/carbon-charts/issues/827

Thanks!

**Comment by mjabbink** [comment]
Excellent exploration @DanaPaslaru

**Review by sophiiae — APPROVED** [review]

**Review by cal-smith — DISMISSED** [review]

**Review by natashadecoste — CHANGES_REQUESTED** [review]
the only thing is that the legend hover doesnt affect the lines  (reducing the opacity)

**Review by cal-smith — APPROVED** [review]

**Review by sophiiae — DISMISSED** [review]

**Review by sophiiae — APPROVED** [review]

**Review by cal-smith — APPROVED** [review]

**Pull request merged**